includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -45,6 +45,8 @@ protected function enqueue_gtag_script() {
 			// If the gtag script is already registered in the Analytics module, then we need to add G-<MEASUREMENT_ID> configuration only.
 			$config = sprintf( 'gtag("config", "G-%s");', esc_js( $this->tag_id ) );
 			wp_add_inline_script( 'google_gtagjs', $config );
+
+			$this->add_inline_ads_conversion_id_config();
```

Comment:
I don't think this part should be here:
* If the `Analytics\Web_Tag` class will output the GA snippet (this conditional), that class already ads the conversion ID as needed.
* Otherwise, the `Analytics_4\Web_Tag` class calls the parent method (below), which already includes this anyway.

So I don't think this class needs to be changed.